### PR TITLE
Social home page

### PR DESCRIPTION
**Pull request by santhoshpemmaka**
Implemented home page for social media app.

Files to check(https://github.com/santhoshpemmaka/NamasteSocialApp/pull/1/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=No+extension&show-deleted-files=false&show-viewed-files=false)

image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 import {Mock} from "./Mock";
 import {makeServer} from "./server";
+
 makeServer();
 ReactDOM.render(
 	<React.StrictMode>
-		<Mock />
-		{/* <App /> */}
+		<Router>
+			{/* <Mock /> */}
```
Comment (iRohitGaur) [inline src/index.js]: suggestion: remove commented code

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by amartya-rs — APPROVED** [review]
Code looks fine to me. Good to merge.

File: src/index.js (hunk 1 above)
Comment (santhoshpemmaka) [reply src/index.js]: I removed it. thanks

**Pull request merged**